### ISSUE TITLE
install fails when a target 'path' specifies an empty directory

### ISSUE THREAD

**Issue by gaopin1534** [yonaskolb/Mint#114]
Hi I found a bit unkind behavior, I don't know any better way so far tho.
when a library specifies a empty directory as a target path in a Package.swift, the installation fails with an error message `Package.swift not found`

This caused mint ver. 0.11.2, swift ver. 4.1.2

my `Package.swift` was like below

```
import PackageDescription

let package = Package(
    name: "Example",
    products: [
        // Products define the executables and libraries produced by a package, and make them visible to other packages.
        .library(
            name: "Example",
            targets: ["Example"]),
        .executable(name: "examplegen", targets: ["examplegen"]),
        .library(name: "ExampleGenCore", targets: ["ExampleGenCore"])
        ],
    dependencies: [
        .package(url: "https://github.com/jpsim/SourceKitten.git", from: "0.21.3"),
        .package(url: "https://github.com/kylef/Commander.git", from: "0.8.0")
        // Dependencies declare other packages that this package depends on.
        // .package(url: /* package url */, from: "1.0.0"),
    ],
    targets: [
//         Targets are the basic building blocks of a package. A target can define a module or a test suite.
//         Targets can depend on other targets in this package, and on products in packages which this package depends on.
        .target(
            name: "examplegen",
            dependencies: ["Commander"],
            path: "Sources/examplegen"),
        .target(
            name: "ExampleGenCore",
            dependencies: ["Target", "SourceKittenFramework"],
            path: "Sources/ExampleGenCore"),
        .target(name: "Example",
                path: "Sources/Example"),
        .testTarget(
            name: "ExampleTests",
            dependencies: ["Target"],
            path: "Tests"),
        ]
)
```
This worked properly at my local however failed installation through mint.
This was because, the tests are not implemented and Tests directory was empty.
Then, git doesn't track Tests directory So mint's `swift package package-dump` fails.

I know this behavior seems correct because it is a proper git behavior.
However if the error message were correct, or you anounced this somewhere in the document, my time would be saved!

thanks,

**Comment by yonaskolb** [comment]
Thanks for raising this @gaopin1534. Yeah the error reporting for the package dump isn't great at the moment. This issue is similar https://github.com/yonaskolb/Mint/issues/106

**Comment by yonaskolb** [comment]
Fixed in 0.11.3
